lib/Runtime/Language/ModuleNamespace.h

```diff
@@ -61,7 +61,7 @@ namespace Js
         virtual BOOL DeleteItem(uint32 index, PropertyOperationFlags flags) override { return true; }
         virtual BOOL GetEnumerator(JavascriptStaticEnumerator * enumerator, EnumeratorFlags flags, ScriptContext* requestContext, EnumeratorCache * enumeratorCache = nullptr);
         virtual BOOL SetAccessors(PropertyId propertyId, Var getter, Var setter, PropertyOperationFlags flags = PropertyOperation_None) override { return false; }
-        virtual BOOL GetAccessors(PropertyId propertyId, Var *getter, Var *setter, ScriptContext * requestContext) override { return false; }
+        virtual BOOL GetAccessors(PropertyId propertyId, __out Var *getter, __out Var *setter, ScriptContext * requestContext) override { getter = nullptr; setter = nullptr; return false; }
```

Comment:
Should this be `*getter = nullptr; *setter = nullptr`?

Reply:
Yes, probably, and with a check on whether the pointers passed in are not nullptr before assigning to them.

Reply:
Ideally these should have been statically verified as non-null because we said _Out_ and not _Out_opt_, right?

Reply:
Also, same pattern is in SpreadArgument.h

Reply:
`*getter = nullptr; *setter = nullptr` would indeed be more correct, but the next set of commits avoids the issue entirely by making the function `_Success_(return) _Check_return_`, and not assigning to either, as the result is always false.